packages/native-federation-typescript/src/rpc/wrap-rpc.ts

```diff
@@ -57,14 +56,12 @@ export function wrapRpc<T extends (...args: any[]) => any>(
           resolveResult(message.value as T);
           if (once) {
             // declare below
-            // eslint-disable-next-line @typescript-eslint/no-use-before-define
             removeHandlers();
           }
         } else if (message.type === RpcGMCallTypes.REJECT) {
           rejectResult(message.error);
           if (once) {
```

Comment:
I mentioned this in the other PR: for childProcess, Node supports both `on` and `once` methods to automatically handle these situations. Consider using it to remove this custom handler

Reply:
I think so too